### PR TITLE
[FEATURE] Mention the site settings editor

### PR DESCRIPTION
**Pull request by linawolf**
References: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1026
Releases: main

### CODE REVIEW

**Review by mkiebele — APPROVED** [review]

File: Documentation/Setup/Settings.rst [hunk 1]
```diff
@@ -78,14 +78,48 @@ Find detailed information in :ref:`TYPO3 Explained, Site handling <t3coreapi:sit
...
 
-..  versionadded:: 12.0
-    Starting with TYPO3 v12 site settings can be saved in a file called
-    :file:`config/sites/my-site/settings.yaml`.
-
-Settings that only affect one site can be made in file
-:file:`config/sites/my-site/settings.yaml`. If that is necessary we will refer
-to this file in the documentation and authors of third party extensions might
-also refer you to it. Find detailed information in
+..  versionchanged:: 13.3
```
Comment (brotkrueml) [inline Documentation/Setup/Settings.rst:81]:
```suggestion
..  versionadded:: 13.3
```

File: Documentation/Setup/Settings.rst [hunk 2]
```diff
@@ -78,14 +78,48 @@ Find detailed information in :ref:`TYPO3 Explained, Site handling <t3coreapi:sit
...
+    Starting with TYPO3 v13.3 site settings can also be edited in the site
+    settings editor.
+
+You can edit site settings in the module :guilabel:`Site Management > Settings`.
+Changes are written to file :file:`config/sites/my-site/settings.yaml`. If you
```
Comment (brotkrueml) [inline Documentation/Setup/Settings.rst:86]: Should we also mention the legacy path "typo3conf/sites/my-site/settings.yaml" for better orientiation? Approx. 50% of the installations out there are legacy.

File: Documentation/Setup/Settings.rst [hunk 3]
```diff
@@ -78,14 +78,48 @@ Find detailed information in :ref:`TYPO3 Explained, Site handling <t3coreapi:sit
...
+*   :composer:`typo3/cms-fluid-styled-content`:
+    :ref:`Site set "Fluid Styled Content" <typo3/cms-fluid-styled-content:site-set-fluid-styled-content>`.
+    Commonly used in projects that do not depend on :composer:`bk2k/bootstrap-package`.
+*   :composer:`typo3/cms-felogin`:
+    :ref:`Settings for the "Frontend Login" site set <typo3/cms-felogin:configuration-site-set-settings>`.
+    Used to offer a login for frontend users.
+*   :composer:`typo3/cms-indexed-search`:
+    :ref:`Settings of the site set "Indexed Search" <typo3/cms-indexed-search:site-set-settings>`.
+    Used to display a search box and offer basic indexing of a page without need
+    for a solr core as would be required by :composer:`apache-solr-for-typo3/solr`.
```
Comment (brotkrueml) [inline Documentation/Setup/Settings.rst:112]:
```suggestion
    for a Solr core as would be required by :composer:`apache-solr-for-typo3/solr`.
```

File: Documentation/Setup/Settings.rst [hunk 4]
```diff
@@ -78,14 +78,48 @@ Find detailed information in :ref:`TYPO3 Explained, Site handling <t3coreapi:sit
...
+*   :composer:`typo3/cms-seo`:
+    :ref:`Site sets settings of EXT:seo <typo3/cms-seo:configuration-site-set-settings>`.
+    Used to improve meta tags, page titles and a provides a XML sitemap.
+
+You can define custom site settings in your site package:
+:ref:`TYPO3 Sitepackage Tutorial <t3sitepackage:start>`.
+
+..  todo: link to the site settings page once it is written.
```
Comment (brotkrueml) [inline Documentation/Setup/Settings.rst:120]: Isn't it the link below to the site settings page? So the todo is obsolete?

File: Documentation/Setup/Settings.rst [hunk 5]
```diff
@@ -78,14 +78,48 @@ Find detailed information in :ref:`TYPO3 Explained, Site handling <t3coreapi:sit
...
+    Used to display a search box and offer basic indexing of a page without need
+    for a Solr core as would be required by :composer:`apache-solr-for-typo3/solr`.
+*   :composer:`typo3/cms-seo`:
+    :ref:`Site sets settings of EXT:seo <typo3/cms-seo:configuration-site-set-settings>`.
+    Used to improve meta tags, page titles and a provides a XML sitemap.
+
+You can define custom site settings in your site package:
+:ref:`TYPO3 Sitepackage Tutorial <t3sitepackage:start>`.
+
+..  todo: link to the site settings page once it is written.
```
Comment (linawolf) [inline Documentation/Setup/Settings.rst:120]:
```suggestion
..  todo: link to the site settings page  in the sitepackage tutorial once it is written.
```

File: Documentation/Setup/Settings.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Setup/Settings.rst:86]: We are in the getting started guide here. And if you installed TYPO3 following the installaition chapter you have a composer-based installation.

File: Documentation/Setup/Settings.rst (hunk 4 above)
Comment (linawolf) [reply Documentation/Setup/Settings.rst]: changed the comment. I meand the site settings chapter in the site package tutorial

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**